src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-8130.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2022-11-15'
 version: 8.13.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.13.0.gem'
+features: ["Updated Rack, Redis, and Sidekiq instrumentation"]
```

Comment:
Were there specific new versions supported? Those might be helpful for customers. I think you could also break this up into three different entries, one per library.

Reply:
[Added](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-71fa490ba702e8d76a6fbe656dffad8534dc06177bd2d50b9772027857a5463dR6) the versions and a little more info for Sidekiq